managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -3645,7 +3648,11 @@ public void testDoNotGetOffloadPoliciesMultipleTimesWhenTrimLedgers() throws Exc
         config.setLedgerOffloader(ledgerOffloader);
 
         ledger.internalTrimConsumedLedgers(Futures.NULL_PROMISE);
-        verify(ledgerOffloader, times(1)).getOffloadPolicies();
+        final LedgerOffloader finalLedgerOffloader = ledgerOffloader;
+        Awaitility.await().untilAsserted(() -> {
+            verify(finalLedgerOffloader, atLeastOnce()).getOffloadPolicies();
+            verify(finalLedgerOffloader, atMost(2)).getOffloadPolicies();
```

Comment:
@Technoboy- It's better to add a comment here to clarify why the method `getOffloadPolicies()` should be invoked once or twice.

Reply:
added.